acceptance/acceptance_test.go

```diff
@@ -1299,6 +1308,11 @@ func runWithLog(t *testing.T, cmd *exec.Cmd, out *os.File, tail bool) (string, e
 	return skipReason, <-processErrCh
 }
 
+func logWithDurationSince(t *testing.T, start time.Time, msg string) {
```

Comment:
followup: it eventualy makes sense to create a runner struct and keep information like start time and a log method in it.